gladier_tools/posix/tar.py

```diff
@@ -0,0 +1,47 @@
+from gladier import GladierBaseTool
+
+
+def tar(data):
+    import os
+    import tarfile
+    tar_input = data['tar_input']
+    if '~' in tar_input:
+        tar_input = os.path.expanduser(tar_input)
+
+    tar_output = data.get('tar_output', f'{tar_input}.tgz')
+    with tarfile.open(tar_output, "w:gz") as tf:
+        tf.add(tar_input)
+    return tar_output
+
+
+class Tar(GladierBaseTool):
```

Comment:
I am not sure we agreed on leaving the GBT inside the __init__ files but I am ok either way.